dimelo/load_processed.py

```diff
@@ -393,9 +468,10 @@ def read_vectors_from_hdf5(
             )
             for tup in read_data_list
         ]
+    #  We add region information (start and end; chromosome is already present!)
```

Comment:
I think this comment should also say "add strand info" or something?

Also for future reference: is this a real dataset or something inferred? I assume it's a real one. I'm just thinking ahead to the questions I have about "real datasets" vs. "calculated datasets" and what the differences are.